DigitalLearningSolutions.Web/Views/Supervisor/ReviewCompetencySelfAsessment.cshtml

```diff
@@ -206,21 +206,39 @@ else if (ViewContext.RouteData.Values["viewMode"].ToString() == "View" && Model.
       }
 
       <div class="nhsuk-summary-list__row">
-      <dt class="nhsuk-summary-list__key">
-        Confirmed
-      </dt>
-      <dd class="nhsuk-summary-list__value">
-        @if (Model.SignedOff)
-        {
-          <span class="nhsuk-tag nhsuk-tag--green">Yes</span>
-        }
-        else
-        {
-          <span class="nhsuk-tag nhsuk-tag--red">No</span>
-        }
-      </dd>
+        <dt class="nhsuk-summary-list__key">
+          Reviewed
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Html.Raw(Model.Verified.ToString("dd/mm/yyyy"))<text> </text>@Model.SupervisorName

```

Comment:
The date in here is always displayed as `01/00/0001` for me:
![image](https://user-images.githubusercontent.com/67740339/168978324-09e4e3ee-47fb-4597-b30d-31eb7393acc0.png)